website/data/internals-nav-data.json

```diff
@@ -44,6 +44,10 @@
     "title": "Provider Metadata",
     "path": "provider-meta"
   },
+  {
+    "title": "Metadata Functions",
+    "path": "metadata-functions"
```

Comment:
```suggestion
    "title": "Functions Metadata",
    "path": "functions-meta"
```

Would it make sense to align this more with the item above?

Reply:
Oh I see now it's under _Internals_ - ignore my last note then! 😄

Reply:
I was following the naming pattern for the docs of other commands, but since this now lives under internals, we could rename it.

Reply:
I'd agree that "Functions Metadata" is clearer now that we're out of the context where repeating the command name is useful.